run/django/requirements.txt

```
@@ -1,6 +1,6 @@
-Django==5.1.4; python_version >= "3.10"
-Django==4.2.17; python_version >= "3.8" and python_version < "3.10"
-Django==3.2.25; python_version < "3.8"
+Django==5.1.5; python_version >= "3.10"
+Django==5.1.5; python_version >= "3.8" and python_version < "3.10"
+Django==5.1.5; python_version < "3.8"
```

Renovate continues to bump versions ignoring environment markers. 

Reverts the renovatebot change, but also updates the latest security fix for 4.2
```suggestion
Django==4.2.18; python_version >= "3.8" and python_version < "3.10"
Django==3.2.25; python_version < "3.8"
```